### DISCUSSION TITLE
does it support HDR mode?

### DISCUSSION THREAD

**Discussion by Ludummy** [xanderfrangos/twinkle-tray#650] [Monitor Compatibility Issues]
Twinkle Tray is wonderful! Thank you for such a great app.

But I‘ve been having an issue when I using it , it dosen't work when I turn on HDR,  it works just fine when HDR mode is off.

Was it designed like that? I was using another similar app, WindowsBA, it works whenever HDR if enable.

Sorry i should have posted it in Issues instead of Discussions.

**Comment by xanderfrangos** [comment]
For the vast majority of monitors, no. HDR on Windows is kind of a mess. I'm not sure there will be a universal solution. At best, SDR app brightness may be adjustable in HDR more. Also, I don't have a real HDR monitor to test with. 

See this issue for more details: https://github.com/xanderfrangos/twinkle-tray/issues/97
